### ISSUE TITLE
App crashes when adding to calendar

### ISSUE THREAD

**Issue by friedger** [ligi/PassAndroid#122]
TraceDroid Version: 1.1
TraceDroid extra:
Stacktrace:
```
kotlin.KotlinNullPointerException
at org.ligi.passandroid.actions.AddToCalendar.b(SourceFile:30)
at org.ligi.passandroid.actions.AddToCalendar.a(SourceFile:13)
at org.ligi.passandroid.actions.AddToCalendar$tryAddDateToCalendar$1.onClick(SourceFile:20)
at android.support.v7.app.AlertController$ButtonHandler.handleMessage(SourceFile:157)
at android.os.Handler.dispatchMessage(Handler.java:102)
at android.os.Looper.loop(Looper.java:148)
at android.app.ActivityThread.main(ActivityThread.java:5461)
at java.lang.reflect.Method.invoke(Native Method)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
Log:
0 D: Writing unhandled exception to: /data/user/0/org.ligi.passandroid/files/3.3.0-1480686384012.tracedroid
1 D: current handler class=com.android.internal.os.RuntimeInit$UncaughtHandler
```

**Comment by ligi** [comment]
Thanks - fix is on the way

**Comment by ligi** [comment]
Bugfix will be included in 3.3.2 - thanks for reporting!
